base/boot.jl

```diff
@@ -246,6 +246,9 @@ struct UndefRefError       <: Exception end
 struct UndefVarError <: Exception
     var::Symbol
 end
+struct AssertEgalError     <: Exception
```

Comment:
Weird amount of space here

Reply:
Lines up with the other Exceptions above

Reply:
But not with `UndefVarError` directly above it, or `InterruptException` directly below it...

Reply:
Well, the people who added those are clearly anarchists.